src/test/scala/com/microsoft/hyperspace/index/PartitionedDataHybridScanTest.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index
+
+import org.apache.hadoop.fs.Path
+
+import com.microsoft.hyperspace.Hyperspace
+import com.microsoft.hyperspace.util.FileUtils
+
+class PartitionedDataHybridScanTest extends HybridScanTest with HyperspaceSuite {
```

Comment:
Can we add tests where users specified "basePath"?

Reply:
For example, we could have a test where an index was created with `spark.read.option("basePath", "/tmp").parquet("/tmp/a=1")`, and if a new partition, `a=2` is introduced, data under `a=2` shouldn't be read. (I believe that's the right behavior).

Reply:
I added 2 tests using basePath but not sure if they cover what you meant.